### ISSUE TITLE
Allow custom constructors

### ISSUE THREAD

**Issue by Bouni** [devatherock/drone-yaml-validator#91]
I have yaml files for use with pyyaml where I have a custom constructor `!include` that allows me to include other yaml files.
The validation however fails because of that. Is there a way to allow such custom elements?

**Comment by devatherock** [comment]
@Bouni Apologies for the delayed response. Let me take a look. It'll be helpful if you can provide the contents of a sample yaml file that fails validation

**Comment by Bouni** [comment]
main.yml
```main.yml
---
title: Title of this file
language: german
date: 16.12.2022
draft: true
elements:
  - !include header
  - !include body
  - !include footer
```

header.yml
```header.yml
---
element1: This is the first element of the header
element2: This is the second element of the header
```

main.yml includes the header.yml in this example

**Comment by devatherock** [comment]
@Bouni I think I have addressed this. Validation shouldn't fail anymore when tags like `!include` are present. The fix is present in the `devatherock/drone-yaml-validator:latest` image. If you can test it when you get a chance and confirm that it works as you expect, I'll release it.

**Comment by devatherock** [comment]
Changes released as `devatherock/drone-yaml-validator:3.0.0`
